### PR TITLE
Refresh download page

### PR DESCRIPTION
**Pull request by slawekjaranowski**
Discussed at https://github.com/apache/maven-ear-plugin/pull/78

### CODE REVIEW

**Review by gnodet — APPROVED** [review]

**Comment by gnodet** [comment]
@slawekjaranowski I think the PR missed the removal of https://github.com/apache/maven-plugin-tools/blob/master/src/site/resources/download.cgi ... ?

File: src/site/xdoc/download.xml.vm [hunk 1]
```diff
@@ -23,102 +23,51 @@ under the License.
...
+              <th></th>
+              <th>Link</th>
+              <th>Checksum</th>
+              <th>Signature</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>${project.name} ${project.version} (Source zip)</td>
+              <td><a href="[preferred]maven/plugin-tools/${project.artifactId}-${project.version}-source-release.zip">${project.artifactId}-${project.version}-source-release.zip</a></td>
```
Comment (gnodet) [inline src/site/xdoc/download.xml.vm:54]: Should n't the link be `https://downloads.apache.org/maven/plugin-tools/...` ?

Comment (slawekjaranowski) [reply src/site/xdoc/download.xml.vm:54]: `[preferred]` will be replaced by cgi to cdn address

https://infra.apache.org/release-download-pages.html#custom

so we need cgi also

Comment (gnodet) [reply src/site/xdoc/download.xml.vm:54]: But that was when there were mirrors, which is not the case anymore IIUC, and that's why the pages have to be updated.  Do I miss something ?

Comment (gnodet) [reply src/site/xdoc/download.xml.vm:54]: Ok, I suppose the executable called by the cgi script has been updated to point to `dlcdn.apache.org`.  Let's keep it that way then.

**Pull request merged**